src/cpp/metadata.cpp

```diff
@@ -230,11 +230,9 @@ void MetaData::get_scalar_values(const std::string& name,
         case SRMetadataTypeString:
             throw SRRuntimeException("MetaData.get_scalar_values() "\
                                      "requested invalid MetaDataType.");
-            break;
```

Comment:
I'm assuming that these changes are to remove compiler warnings for unreachable code. Should they all be in a separate PR?